helpers/helpers.py

```diff
@@ -197,7 +205,9 @@ def fetch_first_open_proposal_url(concluded_proposal_title):
         proposals = data.get("data", {}).get("proposals", [])
         if proposals and proposals[0]["title"] == concluded_proposal_title:
             proposal_id = proposals[0]["id"]
-            return f"https://snapshot.org/#/{constants.SNAPSHOT_SPACE}/proposal/{proposal_id}"
+            return (
+                f"{cfg.SNAPSHOT_URL_PREFIX}{cfg.SNAPSHOT_SPACE}/proposal/{proposal_id}"
```

Comment:
prob should use something like https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin from a forgotten or extra "/" doesn't break things